postreise/plot/plot_states.py

```diff
@@ -64,157 +65,225 @@ def get_state_borders():
     return us_states_dat
 
 
-def plot_states(
+def expand_data_source(patches, state2data, key_name):
+    """Add data to a bokeh patch object
```

Comment:
If this is a user facing function, maybe we need to elaborate a little more on what if does in the doc string.

Reply:
It is more a utility function that is used by `add_state_tooltips`, `add_state_colors` and `add_state_legends` to populate the data source of the patches (the states). It seems to me that there is more we can de correlate from map modules besides the states: color bar, legend, lines (AC and DC), LMP, etc. The `expand_data_source`  could take a more general role in the future.